src/dbt/kipptaf/models/students/intermediate/int_students__attendance_interventions.sql

```diff
@@ -82,25 +67,34 @@ select
     sc.commlog_reason,
     sc.absence_threshold,
 
-    c.user_name as commlog_staff_name,
-    c.commlog_notes,
-    c.commlog_topic,
+    c.response as commlog_notes,
+    c.topic as commlog_topic,
     c.call_date as commlog_date,
-    c.commlog_status,
-    c.commlog_type,
+    c.call_status as commlog_status,
+    c.call_type as commlog_type,
+
+    u.user_name as commlog_staff_name,
 
-    if(c.commlog_reason is not null, 'Complete', 'Missing') as intervention_status,
-    if(c.commlog_reason is not null, 1, 0) as intervention_status_required_int,
+    lc.powerschool_school_id as schoolid,
```

Comment:
Added schoolid using DL. ADA is grouped by student/year, and it would be disruptive to disaggregate that by school w/o making an intentional decision. 

Looking at the query, I assume you want to this data to be attached to the school that issued the comm log. Is that right?